10/README.md

```diff
@@ -20,31 +20,114 @@ fast, scalable network applications. Node.js uses an event-driven, non-blocking
 that makes it lightweight and efficient, perfect for data-intensive real-time applications 
 that run across distributed devices.
 
-Usage
+Usage in OpenShift
 ---------------------
 For this, we will assume that you are using the `ubi8/nodejs-10 image`, available via `nodejs:10` imagestream tag in Openshift.
-Building a simple [nodejs-sample-app](https://github.com/sclorg/s2i-nodejs-container/tree/master/10/test/test-app) application
+Building a simple [nodejs-sample-app](https://github.com/sclorg/nodejs-ex.git) application
 in Openshift can be achieved with the following step:
 
-    ```
-    oc new-app nodejs:10~https://github.com/sclorg/s2i-nodejs-container.git --context-dir=10/test/test-app/
-    ```
+```
+oc new-app nodejs:10~https://github.com/sclorg/nodejs-ex.git
+```
 
-The same application can also be built using the standalone [S2I](https://github.com/openshift/source-to-image) application on systems that have it available:
+**Accessing the application:**
+```
+$ oc get pods
+$ oc exec <pod> -- curl 127.0.0.1:8080
+```
 
-    ```
-    $ s2i build https://github.com/sclorg/s2i-nodejs-container.git --context-dir=10/test/test-app/ ubi8/nodejs-10 nodejs-sample-app
-    ```
+Source-to-Image framework and scripts
+---------------------
+This image supports the [Source-to-Image](https://docs.openshift.com/container-platform/3.11/creating_images/s2i.html)
+(S2I) strategy in OpenShift. The Source-to-Image is an OpenShift framework
+that makes it easy to write images that take application source code as
+an input, use a builder image like this Node.js container image, and produce
+a new image that runs the assembled application as output.
+
+In order to support the Source-to-Image framework, there are some interesting scripts inside the builder image:
+
+* The `/usr/libexec/s2i/assemble` script inside the image is run in order to produce a new image with the application artifacts. The script takes sources of a given application and places them into appropriate directories inside the image. It utilizes some common patterns in Node.js application development (see **Environment variables** section below).
+* The `/usr/libexec/s2i/run` script executes the application and is set as a default command in the resulting container image.
```

Comment:
I don't follow is here? Can you please rephrase it?

Reply:
Ok, will try. What about this?
```
* The `/usr/libexec/s2i/run` script is set as a default command in the resulting container image (the new image with the application artifacts). It runs `npm run` for production, or `nodemon` in case `DEV_MODE` is set to `true` (see **Environment variables** section below).
```

Reply:
Changed now, but feel free to provide further feedback if this rewording is not perfect yet.

Reply:
LGTM now.